documentation/src/main/asciidoc/advanced-features.asciidoc

```diff
@@ -19,6 +20,22 @@ SearchFactory searchFactory = fullTextSession.getSearchFactory();
 ----
 ====
 
+[[AccessingSearchIntegrator]]
+=== Accessing the SearchIntegrator
```

Comment:
Yes ideally it should. But let's focus on your fix now.. if you want to change the others too, that's great but keep it as as a separate commit.